apm-sniffer/apm-sdk-plugin/rabbitmq-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/rabbitmq/RabbitMQConsumerInterceptor.java

```diff
@@ -51,7 +51,11 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
         CarrierItem next = contextCarrier.items();
         while (next.hasNext()) {
             next = next.next();
-            next.setHeadValue(properties.getHeaders().get(next.getHeadKey()).toString());
+            if (properties != null) {
+                next.setHeadValue(properties.getHeaders().get(next.getHeadKey()).toString());
```

Comment:
I think, you may still get NPE, when
1. Headers don't exist.
1. Key doesn't exist in header.

Reply:
Yes~